atd-vze/src/queries/crashes.js

```diff
@@ -118,6 +118,21 @@ export const GET_CRASH = gql`
   }
 `;
 
+export const GET_CRASH = gql`
+  query CrashDetails($crashId: Int!) {
+    crashes_by_pk(crash_id: $crashId) {
+      crash_id
+      address_primary
+      address_secondary
+      crash_injury_metrics {
+        cris_fatality_count
+        sus_serious_injry_count
+        years_of_life_lost
+      }
```

Comment:
the only way i could think of incrementally working on the crash details page is to create another query and be using both for rendering the app, and with each PR this query will grow and handle more of the querying

Reply:
this makes sense!